<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitInfoWidget</class>
 <widget class="QWidget" name="GitInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="relativePathLabel">
       <property name="text">
        <string>Relative Path: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="relativePathEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="diffToLabel">
       <property name="text">
        <string>Diff to: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="diffToComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="customDiffHashLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="customDiffHashLabel">
       <property name="text">
        <string>Custom Diff Hash:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customDiffHashEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="hashLabel">
     <property name="text">
      <string>Hash: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="diffLabel">
     <property name="text">
      <string>Diff</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="diffText"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
